design/AstroMancer Cards.md

```diff
@@ -0,0 +1,21 @@
+##AstroMancer Cards
+
+###AstroMancer Cards can be played for 2 Mana
```

Comment:
Are you supposed that each character will use a separate resource pool, instead of a shared pool like HoG?

Reply:
I am supposing this. Yes. But I could reimagine the characters and make them share a pool.